### DISCUSSION TITLE
not supported calendar message.ics

### DISCUSSION THREAD

**Discussion by kommaRob** [spatie/icalendar-generator#61] [Bugs]
I have a issue so i tried to simplify it by using this example code:

```
$calendarData = Calendar::create('Laracon online')
            ->event(CalenderEvent::create('Creating calender feeds')
                ->startsAt(new DateTime('6 March 2019 15:00'))
                ->endsAt(new DateTime('6 March 2019 16:00'))
            )
            ->get();
```

In the code i named the calender "**calendar.ics**" and when i view the e-mail in Apple Mail on my iPhone, the attachment has that name. But viewed in Outlook mail client the attached .ics file has the name: "**not supported calendar message.ics**"

**Comment by rubenvanassche** [comment]
I don't have access to Outlook mail but a quick Google search on `not supported calendar message.ics` shows that your MIME types should be correct, maybe that could be the problem?
